core/trino-main/src/main/java/io/trino/metadata/MetadataManager.java

```diff
@@ -1423,6 +1458,19 @@ public void setMaterializedViewProperties(Session session, QualifiedObjectName v
         metadata.setMaterializedViewProperties(session.toConnectorSession(catalogHandle), viewName.asSchemaTableName(), properties);
     }
 
+    private static boolean isExternalInformationSchema(CatalogHandle catalogHandle, Optional<String> schemaName)
+    {
+        return isExternalInformationSchema(catalogHandle, schemaName.orElse(null));
+    }
+
+    private static boolean isExternalInformationSchema(CatalogHandle catalogHandle, @Nullable String schemaName)
+    {
+        return switch (catalogHandle.getType()) {
+            case INFORMATION_SCHEMA, SYSTEM -> false;
```

Comment:
I see there already has a method `isInternal` to check, maybe we can use it as it is possible added more `CatalogHandleType`?
Is all the `information_schema` is the name equals to the text "information_schema"? will it require connector to provided an extra name by themselves in the future?

Reply:
`information_schema ` is defined in SQL standard. If remote databases use different name, it should be handled in connectors instead of engine.